back-end/aws-cloud9-api/src/main/java/com/serverless/GetUserFormsHandler.java

```diff
@@ -0,0 +1,47 @@
+package com.serverless;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.Map;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.serverless.dal.Form;
+import com.serverless.dal.FormDBTable;
+import java.util.Collections;
+import java.util.Map;
+import java.util.List;
+
+public class GetUserFormsHandler implements RequestHandler<Map<String, Object>, ApiGatewayResponse>{
+	private static final Logger LOG = LogManager.getLogger(Handler.class);
+
+	@Override
+	public ApiGatewayResponse handleRequest(Map<String, Object> input, Context context) {
+		try {
+			@SuppressWarnings("unchecked")
+			Map<String,String> pathParameters =  (Map<String,String>)input.get("pathParameters");
+	        String userId = pathParameters.get("id");
+
+			return ApiGatewayResponse.builder()
+      				.setStatusCode(200)
+      				.setObjectBody(new FormDBTable().getForms(userId))
```

Comment:
Pobranie odpowiednich testów zrób przed returnem. Jeśli nie znaleziono żadnych testów, to fajnie by było, żeby był jakiś komunikat o tym, albo w ogóle dać 404

Reply:
A nie wolimy po prostu wtedy zwracać pustej tablicy, tak jak jest teraz?